bitrise.yml

```diff
@@ -1437,7 +1437,7 @@ workflows:
             PROJECT="/Users/vagrant/git/firefox-ios/Client.xcodeproj"
             SCHEME="Fennec_Enterprise"
             SDK="iphoneos"
-            DESTINATION="generic/platform=iOS"
+            DESTINATION="platform=iOS Simulator,name=iPhone 16,OS=18.0"
```

Comment:
If we want to build for iOS 18.0, we need to select a destination device that has been installed on Bitrise's runner. Bitrise does not have iPhone 15 running OS 18.0.